### PR TITLE
Fast EOF

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

**Comment by e5l** [comment]
Hi @qwwdfsad, thanks for the PR. As mentioned before, this change won't prevent filling the internal `Input` buffer. Could you add some measurements to compare the change?

**Comment by qwwdfsad** [comment]
Yeah, that's why I haven't touched our primitives like `copyTo`.
This change is more about readability and the code being straightforward, not about performance

**Comment by e5l** [comment]
LGTM

File: core/commonMain/src/kotlinx/io/Input.kt [hunk 1]
```diff
@@ -214,7 +214,9 @@ public abstract class Input : Closeable {
      * performance-critical code.
      */
     @Suppress("NOTHING_TO_INLINE")
```
Comment (e5l) [inline core/commonMain/src/kotlinx/io/Input.kt:216]: Suppress can be removed.

**Pull request merged**